fixit/rules/no_namedtuple.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, Sequence, Tuple
+
+import libcst as cst
+from fixit.common.base import CstLintRule
+from fixit.common.utils import InvalidTestCase as Invalid, ValidTestCase as Valid
```

Comment:
Even though this works, the convention is to import those helpers from `fixit` to be simpler and consistent to other rules.
https://fixit.readthedocs.io/en/latest/test_a_lint_rule.html#Writing-Unit-Tests